### ISSUE TITLE
Host Kyma CLI documentation on the website (POC)

### ISSUE THREAD

**Issue by bszwarc** [kyma-project/cli#279]
**Description**

Kyma CLI is gaining interest, but the only documentation we have is here:  https://github.com/kyma-project/cli/tree/master/docs/gen-docs and the readme. It is mentioned in the installation guides and further throughout the Kyma docs, but we miss a dedicated section. 
Documentation is now kept in a separate repository, so we need to figure out how to host it on the website and if it is possible if the CLI docs are NOT in `kyma/docs`. 

Where to put it?
A good place for Kyma CLI docu would be under the main Kyma node.

What goes in there:
- What Kyma CLI is
- Arch doc
- Examples of use
- Documentation for commands (we can use the auto-generated command documentation).

Concerns:
- Is it possible to take the docs out of `kyma/cli` repo and feed them to the website? We would like to avoid copying the documentation and maintain it in two separate places.

**Reasons**

Have the CLI docu on the website, available for the users to browse and refer to.

**Comment by bszwarc** [comment]
This is still valid.

**Comment by bszwarc** [comment]
I consulted this issue with Wookiees. Picking up docs for the website from sources other than Kyma & community is somewhere in the pipeline, but not a high priority right now. 


tl; dr:

Let's have proper CLI documentation in Kyma docs (it's kyma CLI after all), generate our docs there and for the CLI repo - leave a short README file with all the necessary links. 

More information 
What we can do to is:
* add a `cli` folder under `kyma/docs` to render a separate component in the component list.
**NOTE** We can have it under `kyma` but that would require having a `cli` doc type, which we don't have and it doesn't make much sense to introduce it. That's why it's easier to have it as a separate node in the component hierarchy. 
* add documents there, describing CLI features, arch & other things

FOR THE GENERATED DOCS we have 2 possibilities:

1. change the output folder for the generated docs to generate them directly to e.g. `kyma/docs/cli/generated` or something similar. Then we would have a short README.md in the main CLI repo with all the necessary links and the majority of docs in Kyma. 
2. leave the docs where they are and have a link in Kyma docs.

I would go with the first solution for this point.

**Comment by bszwarc** [comment]
Prepare the content for the `website as a template` poc/spike
See also: 
https://github.com/kyma-project/website/issues/493
https://github.com/kyma-project/cli/issues/278 
https://github.com/kyma-project/kyma/issues/8190

**Comment by bszwarc** [comment]
If there are any follow up things to do on this task, we will create a separate PR. I wanted to merge it today so that Maciej can work on the website part using the structure.
